indra/llfilesystem/lldir_win32.cpp

```diff
@@ -390,6 +390,15 @@ bool LLDir_Win32::fileExists(const std::string &filename) const
 	}
 }
 
+// virtual
+bool LLDir_Win32::skinExists(const std::string& subdir, const std::string &skin) const
+{
+    if (subdir == LLDir::XUI && skin == "en")
```

Comment:
Why 'en' only if you are making a copy of all skins?

Reply:
Because we use this check for 2 skins only, 'en' and 'en-us'
For xui we provide 'en' but not 'en-us'
Thus this check is very simple